akita_inu_asa_utils.py

```diff
@@ -47,8 +52,26 @@ def compile_program(client, source_code, file_path=None):
         dump(base64.b64decode(compile_response['result']), 'build/' + file_path)
 
 
+# read user local state
+def read_local_state(client, addr, app_id):
+    results = client.account_info(addr)
+    print(results)
+    if len(results['apps-local-state']):
+        local_state = results['apps-local-state'][0]
+        for index in local_state:
+            if local_state[index] == app_id:

```

Comment:
if local_state[index] == app_id and if 'key-value' in local_state.keys():

Reply:
yup! Depending on the condition of the local state. If I remember right, local_state[index] may contain the local schema, however does not guarantee that the 'key-value' key exists, which contains local_state variables